queries.sql

```diff
@@ -1,3 +1,30 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon"
+SELECT * from animals WHERE name LIKE '%mon%';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts
+SELECT name from animals
+  WHERE (neutered = 't') 
+  AND escape_attempts >= 3;
```

Comment:
- [ ] For all animals that are neutered and have less than 3 escape attempts. It will be better to use `true` and not `t` because t is not a column type of a table in pastgres

  ```sql
  SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';
  ```